packages/data-point/lib/entity-types/entity-hash/resolve.test.js

```diff
@@ -29,24 +29,24 @@ beforeAll(() => {
   resolveReducerBound = helpers.createReducerResolver(dataPoint)
 })
 
-describe('entity.hash.resolve', () => {
-  test('entity.hash - only process Plain Objects', () => {
-    return transform('hash:asIs', [testData])
-      .catch(result => {
-        return result
-      })
+describe('Hash entity type checking', () => {
+  function resolveInvalid (entity, data) {
+    return dataPoint
+      .resolve(entity, data)
+      .catch(err => err)
       .then(result => {
         expect(result).toBeInstanceOf(Error)
-        expect(result.message).toMatchSnapshot()
-      })
-  })
-
-  test('entity.hash - throw error if value is not object', () => {
-    return transform('hash:noValue', null)
-      .catch(e => e)
-      .then(result => {
         expect(result).toMatchSnapshot()
       })
+  }
+  test('should throw error from default outputType reducer when input is not valid', () => {
```

Comment:
Shouldn't this be `should throw error from default outputType reducer when output is not valid`?

Reply:
good catch - fixed this along with a similar mistake in `entity-collection/resolve.test.js`